<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>593</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>2</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="6" rowspan="8" colspan="2">
     <widget class="QListWidget" name="listWidget_2"/>
    </item>
    <item row="8" column="6">
     <widget class="QPushButton" name="neu_2">
      <property name="text">
       <string>neu2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="start_2">
      <property name="text">
       <string>start 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSpinBox" name="h"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="start">
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QPushButton" name="stop">
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" rowspan="2" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>132</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4" rowspan="8" colspan="2">
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QPushButton" name="resume">
      <property name="text">
       <string>resume</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="entfernen">
      <property name="text">
       <string>entfernen</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>Neuer Timer</string>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="QLabel" name="timerlabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="neu">
      <property name="text">
       <string>neu</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="pause">
      <property name="text">
       <string>pause</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSpinBox" name="m"/>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QSpinBox" name="a_m"/>
    </item>
    <item row="2" column="0">
     <widget class="QSpinBox" name="a_h"/>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="s">
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="8" column="7">
     <widget class="QPushButton" name="entfernen_2">
      <property name="text">
       <string>entferne</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="a_s"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
